src/basicfuns.jl

```diff
@@ -94,6 +94,16 @@ logit(x::Real) = log(x / (one(x) - x))
 """
 $(SIGNATURES)
 
+Return `log(cosh(x))`.
```

Comment:
Mention what's the difference with just calling `log(cosh(x))` (i.e. precision, `abs`, etc.)?

Reply:
```suggestion
Return `log(cosh(x))`, carefully evaluated without intermediate calculation of `cosh(x)`.

The implementation ensures `logcosh(-x) = logcosh(x)`.
```

Reply:
I extended the docstring a bit :slightly_smiling_face: